app/views/search/_result.html.erb

```diff
@@ -28,6 +28,12 @@
           <%= render partial: 'shared/contributors', locals: { contributors: result[:contributors] } %>
         </ul>
       <% end %>
+
+      <% if result[:summary].present? %>
+        <div class="result-summary truncate-list">
```

Comment:
Non-blocking comment: I see that we're using a `div` element here, while the geodata partial uses a paragraph element for the summary. I don't know if that's meaningful, and is ultimately a question for Dave I think, but there's a part of me that worries about unintended divergence in the templates that provide each instance.

Reply:
GeoData needs to be completely reworked once we stabilize USE. I think consistency within the USE partials is the main focus right now, and yeah I'm letting Dave determine ultimate markup.